api/comment_threads.rb

```diff
@@ -54,8 +54,9 @@
   filter_blocked_content params["body"]
   thread.update_attributes(params.slice(*%w[title body pinned closed commentable_id group_id thread_type]))
 
-  if params["user_id"] and value_to_boolean(params["read"])
-    user = User.only([:id, :username, :read_states]).find_by(external_id: params["user_id"])
+  # user_id is thread user while requested_user_id is user requested thread update
```

Comment:
Sentence is slightly had to follow for me. 
Just to make sure I understood this,

user_id is the owner for a thread, requested_user_id the user requesting to update said thread